packages/common-all/src/types/configs/publishing/giscus.ts

```diff
@@ -0,0 +1,77 @@
+// https://github.com/giscus/giscus-component/blob/main/react/src/lib/types.ts
+
+// non-trivial / const related type definitions are breaking JSON schema generation
```

Comment:
There are some generic typing on Giscus' end that our JSON schema generator fails.
Leaving this to be fixed on the next phase.